helix-front/pom.xml

```diff
@@ -92,13 +94,13 @@
 
           <execution>
             <id>build</id>
+            <phase>process-test-resources</phase>
             <goals>
-              <goal>npm</goal>
+              <goal>yarn</goal>
             </goals>
             <configuration>
-              <arguments>run-script build</arguments>
+              <arguments>build</arguments>
```

Comment:
The syntax is different between yarn and npm.  https://classic.yarnpkg.com/en/docs/cli/run

Ah! it looks like this should be `run build`.  Will fix.